package.json

```diff
@@ -95,6 +95,7 @@
     "@osu-cass/tslint-config": "^2.0.0",
     "@types/body-parser": "^1.17.0",
     "@types/elasticsearch": "^5.0.26",
+    "@types/event-stream": "^3.3.34",
```

Comment:
Not sure why this needs to be here, especially after the NPM vulnerability.

Reply:
Do you think we should remove this before merging in?